pkg/kv/dist_sender.go

```diff
@@ -59,6 +60,36 @@ const (
 	defaultSenderConcurrency = 500
 )
 
+var (
+	// DistSender Metrics
+	metaTotalSends         = metric.Metadata{Name: "distsender.totalSends"}
```

Comment:
can you add help strings to all of those to describe what they are.
Maybe also a longer comment in this block explaining the relationship between these metrics.

For now, I can't tell what they mean, so I can't tell if what the names should be.